src/org/anjocaido/groupmanager/GroupManager.java

```diff
@@ -185,18 +186,21 @@ public void onEnable(boolean restarting) {
 			selectedWorlds = new HashMap<>();
 			lastError = ""; //$NON-NLS-1$
 
+			if (!restarting) {

```

Comment:
This section is still not correct. Things are out of order. Like loading GlobalGroups. This whole section change (other than the logger changes) should probably be scrapped.